mods/ra/maps/soviet-07/soviet07.lua

```diff
@@ -239,7 +239,9 @@ IntroSequence = function()
 		timerStarted = true
 	end)
 	Trigger.OnAllKilled(StartingUnits, function()
```

Comment:
The point of it was that you need a human unit to interact with the security system and can't use an animal for that job. I'll remove that Trigger if the majority wants it gone.